config_manager/include/config_manager.h

```diff
@@ -0,0 +1,16 @@
+//
+// Created by viacheslav on 11.04.23.
```

Comment:
Git stores the commit author, so there is no need for additional lines with author name in each file.